.travis/docker_run.sh

```diff
@@ -4,14 +4,35 @@ cd "$(dirname $0)/.."
 CHROOT_PATH="/opt/rootfs"
 MACHINEKIT_PATH="/usr/src/machinekit"
 TRAVIS_PATH="$MACHINEKIT_PATH/.travis"
+CONTAINER="kinsamanka/mkdocker"
 
+cmd=${CMD}
+if [ ${CMD} == "run_tests" ];
+then
+    cmd=build_rip
+fi
+
+# run build step
 docker run \
-    --privileged=true \
     -v $(pwd):${CHROOT_PATH}${MACHINEKIT_PATH} \
     -e FLAV="${FLAV}" \
-    -e MK_DIR=${TAG}_${CMD} \
     -e JOBS=${JOBS} \
     -e TAG=${TAG} \
-    -e ROOT=${CHROOT_PATH} \
-    kinsamanka/mkdocker:${TAG} \
-    ${CHROOT_PATH}${TRAVIS_PATH}/${CMD}.sh
+    -e CHROOT_PATH=${CHROOT_PATH} \
+    -e MACHINEKIT_PATH=${MACHINEKIT_PATH} \
+    -e TRAVIS_PATH=${TRAVIS_PATH} \
+    ${CONTAINER}:${TAG} \
+    ${CHROOT_PATH}${TRAVIS_PATH}/${cmd}.sh
+
+# tests are run under a new container instead of chrooting
+# this will allow us to run docker without using privileged mode
+if [ ${CMD} == "run_tests" ];
+then
+    # create container using RIP rootfs
+    docker build -t mk_runtest .travis/mk_runtests
+    
+    # run regressions
+    docker run \
+        -e MACHINEKIT_PATH=${MACHINEKIT_PATH} \
+        --rm=true mk_runtest /run_tests.sh
```

Comment:
Could this be done with `proot -R` instead of using the .tar.gz and building a docker container?